package.json

```diff
@@ -10,6 +10,8 @@
   },
   "dependencies": {
     "lodash": "^4.17.4",
+    "nodeless-trakt": "^1.0.0",
+    "query-string": "^4.3.4",
```

Comment:
is it possible to use this instead ? https://nodejs.org/api/querystring.html
not sure if the transpile process would automatically include that builtin package... I think it does